src/Features/CSharp/Portable/ExtractMethod/CSharpMethodExtractor.CSharpCodeGenerator.cs

```diff
@@ -791,6 +793,19 @@ protected SyntaxToken CreateMethodNameForStatementGenerators(bool localFunction)
 
                 return SyntaxFactory.Identifier(nameGenerator.CreateUniqueMethodName(scope, "NewMethod"));
             }
+
+            protected override async Task<(ExpressionBodyPreference expressionBodiedMethod, ExpressionBodyPreference expressionBodiedLocalFunction, bool staticLocalFunction)> StaticLocalFunctionAndExpressionBodyPreferenceAsync(
```

Comment:
```suggestion
            protected override async Task<(ExpressionBodyPreference expressionBodiedMethod, ExpressionBodyPreference expressionBodiedLocalFunction, bool staticLocalFunction)> GetStaticLocalFunctionAndExpressionBodyPreferenceAsync(
```

Reply:
It might also be wise to either just use an OptionSet here or something other than a tuple. The risk of a tuple with two ExpressionBodyPreferences is somebody might cross them when handling the tuple and you end up with really confusing bugs.